ui/DesignSystem/Primitive/Avatar.svelte

```diff
@@ -119,11 +122,7 @@
 
 <div data-cy={dataCy} class={`container ${size}`} {style}>
   {#if imageUrl}
-    <img
-      class={`image ${avatarClass}`}
-      src={imageUrl}
-      alt="user-avatar"
-      onerror="this.style.display='none'" />
```

Comment:
Why aren't we keeping the `onerror`?

Reply:
Because it's [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Browser_compatibility).